wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -355,3 +356,94 @@ def test_items_dict_1_included_keys(self):
         self.assertTrue("facebook_shares" in cleaned_postmeta)
         self.assertTrue("pinterest_shares" in cleaned_postmeta)
         self.assertTrue("twitter_shares" in cleaned_postmeta)
+
+
+class TestWordpressItemPrefilterConfig(TestCase):
+    def test_prefilter_content_default(self):
+        # The expected output should be transformed after passing through the
+        # the default prefilters
+        node = {"content:encoded": "foo bar baz"}
+        wordpress_item = WordpressItem(node, "")
+        output = wordpress_item.prefilter_content(wordpress_item.raw_body)
+        self.assertEqual(output, "<p>foo bar baz</p>\n")
+
+
+class TestWordpressPrefilterDefaults(TestCase):
+    def test_default_prefilters(self):
+        defaults = default_prefilters()
+        self.assertIsInstance(defaults, list)
+        self.assertTrue(len(defaults), 4)
+        self.assertEqual(
+            defaults[0]["FUNCTION"],
+            "wagtail_wordpress_import.prefilters.linebreaks_wp",
+        )
+        self.assertEqual(
+            defaults[1]["FUNCTION"],
+            "wagtail_wordpress_import.prefilters.transform_shortcodes",
+        )
+        self.assertEqual(
+            defaults[2]["FUNCTION"],
+            "wagtail_wordpress_import.prefilters.transform_inline_styles",
+        )
+        self.assertEqual(
+            defaults[3]["FUNCTION"],
+            "wagtail_wordpress_import.prefilters.bleach_clean",
+        )
+
+
+def foo_filter(content, options):
+    return content, options
+
+
+@override_settings(WAGTAIL_WORDPRESS_IMPORT_PREFILTERS=[])
```

Comment:
**Suggestion**: Since these test case methods are all testing related things, it would make sense for them to be in the same TestCase class. Isn't `@override_settings` useable as a method decorator?

Reply:
I don't think they can be combined because each test initialises WordpressItem where the config is read each time.

Reply:
Oh. Again, I'm not sure how that is causing a problem, because there are no module-level function calls in `wordpress.py`. Still, if it doesn't work as a single class, it is okay like this.